subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junitplatform/JUnitPlatformIntegrationTest.groovy

```diff
@@ -383,8 +385,11 @@ public class StaticInnerTest {
 
             import java.util.concurrent.*;
             import org.junit.jupiter.api.*;
+            import org.junit.jupiter.api.parallel.*;
             import static org.junit.jupiter.api.Assertions.*;
+            import static org.junit.jupiter.api.parallel.ExecutionMode.CONCURRENT;
 
+            @Execution(CONCURRENT)
```

Comment:
These changes are necessary because JUnit 5.4.0 changed the default execution mode from `CONCURRENT` to `SAME_THREAD`.